xcell/mappers/mapper_dummy.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, config):
 
     def _check_dtype(self):
         dtypes = ['galaxy_density', 'galaxy_shear',
-                  'cmb_convergence', 'cmb_tSZ']
+                  'cmb_convergence', 'cmb_tSZ', 'generic']
         if self.dtype not in dtypes:
             raise ValueError(f'Tracer type {self.dtype} not implemented.')
```

Comment:
Since you will be using this in `cov.py`, I would update it to NotImplementedError.

Reply:
Done